setup.cfg

```diff
@@ -23,6 +23,7 @@ setup_requires = setuptools_scm[toml] >= 3.4.1
 [options.extras_require]
 testing =
 	# upstream
+	coverage < 5
```

Comment:
More importantly, what's this line about? Is it to add coverage? To limit coverage? I do try to avoid pinning dependencies without a reason to pin (and a comment to explain why). Looking quickly at coverage, this is a downgrade from what's currently used. Is it necessary to support this feature? If so, there should be a comment explaining that.